src/main.ts

```diff
@@ -59,7 +60,7 @@ async function bootstrap(): Promise<void> {
   if (isApiKeyStrategyConfigured) {
     document.security = [{ api_key: [] }]; // Apply the API key globally to all operations
   }
-
+  fs.writeFileSync('./swagger-spec.json', JSON.stringify(document));
```

Comment:
I was thinking along the lines of having a separate script that would run in the actions and not touch a developers HD.

This is probably OK and is less work. As long as we commit the file we should be good. I can't see it getting out of date when its updated on start

Reply:
But a script might be a cleaner solution IMO. It will ensure to upload the corresponding swagger file for each build rather than depend to commit it.

Reply:
@prashantasdeveloper you can look at the `repl.ts` + package.json script using it as an example

Reply:
I am certainly appreciating the simplicity now.

For this to run in CI we need to figure out a way to not need a Polymesh node without making a mess out of our config